qemu/qemus390x_test.go

```diff
@@ -30,7 +30,7 @@ var (
 	deviceNetworkPCIStringMq       = "-netdev tap,id=tap0,vhost=on,fds=3:4 -device driver=virtio-net-ccw,netdev=tap0,mac=01:02:de:ad:be:ef,bus=/pci-bus/pcie.0,addr=ff,mq=on"
 	deviceSerialString             = "-device virtio-serial-ccw,id=serial0"
 	deviceVSOCKString              = "-device vhost-vsock-ccw,id=vhost-vsock-pci0,guest-cid=4"
-	deviceVFIOString               = "-device vfio-ccw,host=02:10.0"
+	deviceVFIOString               = "-device vfio-ccw,host=02:10.0,x-pci-vendor-id=0x1234,x-pci-device-id=0x5678"
```

Comment:
@alicefr Do these changes make sense for s390x?  This is just a test case but I was just wondering whether these new ids will be used on s390x.

Reply:
I know the parameter is ok for vfio-pci, but I do not know if it is ok for `s390`, it will be helpful if someone helps

Reply:
@Ace-Tang  vfio-ccw does not support it.

Reply:
no, those flags are not supported by virtio-ccw. See:
```sh
$ qemu-system-s390x -device vfio-ccw,help
vfio-ccw.devno=str (Identifier of an I/O device in the channel subsystem, example: fe.1.23ab)
vfio-ccw.sysfsdev=str
```

Reply:
I also need to check the host flag, but it's not related to this PR